less/index.less

```diff
@@ -50,10 +50,267 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+  height: 100%;
 }
 
+// Variables
```

Comment:
Your variables look great! Good job on the stretch goal with adding in a google font 👍